.eslintrc.json

```diff
@@ -1,11 +1,18 @@
 {
   "root": true,
-  "ignorePatterns": ["projects/**/*"],
+  "ignorePatterns": [
+    "projects/**/*"
+  ],
```

Comment:
Please run `yarn prettier:write` see [package.json](https://github.com/cloudnc/ngx-favicon/blob/master/package.json#L10). That'll run prettier and save a lot of diffs on the PR. Thanks